docs/maui_support.md

```diff
@@ -0,0 +1,46 @@
+# Migrating from Xamarin.Forms to .NET MAUI
+
+upgrade-assistant include preview support for Xamarin.Forms to .NET MAUI migration. Please note this tool only has preview support and more features coming soon. This tool helps with the migration progress but you will still require to make manual changes to the solution to make it run successfully with .NET MAUI.
+
+## Supported Project types and minimum requirements
+
+- Xamarin.Forms app using Xamarin.Forms 5.0 or higher
+  - ✅ iOS project
+  - ✅ Android project
+  - ❌ UWP project 
+  - ❌ iOS App extension and other extension types for example Share Extension project
+
+- Complete environment setup for .NET MAUI including all necessary workloads installed. Confirm that your environment can build and run a blank .NET MAUI template app. Be sure to follow all steps mentioned in the [Getting Started](https://github.com/dotnet/maui/wiki#getting-started). 
+- Use version control, create a backup of current code before starting the upgrade process
```

Comment:
"We recommend using version control to create a backup of your current codebase before starting the upgrade process"?